model/task_lifecycle.go

```diff
@@ -943,13 +945,23 @@ func updateMakespans(b *build.Build, buildTasks []task.Task) error {
 	return errors.WithStack(b.UpdateMakespans(depPath.TotalTime, CalculateActualMakespan(buildTasks)))
 }
 
+type buildStatus struct {
+	status              string
+	allTasksBlocked     bool
+	allTasksUnscheduled bool
+}
+
 // getBuildStatus returns a string denoting the status of the build and
 // a boolean denoting if all tasks in the build are blocked.
-func getBuildStatus(buildTasks []task.Task) (string, bool) {
+func getBuildStatus(buildTasks []task.Task) buildStatus {
 	// Check if no tasks have started and if all tasks are blocked.
 	noStartedTasks := true
 	allTasksBlocked := true
+	allTasksUnscheduled := true
 	for _, t := range buildTasks {
+		if !t.IsUnscheduled() {
```

Comment:
Preference for just having this function be IsScheduled -- more intuitive to check on the positive; and then the function comment makes a bit more. sense, and this line can just check if t.IsScheduled()

Reply:
I think I prefer isUnscheduled because it mimics blocked. It also echos the UI "unscheduled" state. 
![image](https://user-images.githubusercontent.com/13104717/200603943-a75199a2-ca88-43a3-b863-88c50ada0612.png)

I see how the comment left something to be desired. I updated to what is hopefully more intuitive. Does it feel better with the comment update?